.github/workflows/submitAndMatch.yml

```diff
@@ -0,0 +1,60 @@
+name: submitAndMatch
+
+on:
+  push:
+    branches:
+      - 'integration'
+      - 'rel-v8r0'
```

Comment:
By adding these branches, we are basically restricting the run of this workflow only after the merge is made. 
If a developer wants to run this, these branches should be removed, and the secrets added to the fork.